config/rootfs/debos/scripts/install-firmware.sh

```
@@ -0,0 +1,27 @@
+#!/bin/sh
+
```

To avoid silent failures:
```suggestion

set -e
```